samplesheets/tests/test_rendering.py

```diff
@@ -202,6 +203,35 @@ def test_get_study_tables_cache(self):
         study_tables = self.tb.get_study_tables(self.study)
         self.assertEqual(study_tables, cache_item.data)
 
+    @override_settings(SHEETS_ENABLE_STUDY_TABLE_CACHE=False)
+    def test_get_study_tables_config(self):
+        """Test get_study_tables() with SHEETS_ENABLE_STUDY_TABLE_CACHE=False"""
+        tables = self.tb.get_study_tables(self.study)
+        t_field = tables['study']['top_header'][0]
+        self.assertEqual(t_field['value'], 'Source')
```

Comment:
We don't need to assert the first top header being called `Source`, that is a given in all of our ISA-Tabs.